### PR TITLE
Add Group to puml

### PR DESCRIPTION
**Pull request by martout2002**
added group to two of the file where the person object is mentioned in the .puml files

### CODE REVIEW

**Review by wenjebs — COMMENTED** [review]
looks good

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -18,4 +18,5 @@ Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+Person *--> Group
```
Comment (Hoodini231) [inline docs/diagrams/BetterModelClassDiagram.puml:21]: good addition!

**Review by Hoodini231 — APPROVED** [review]
LGTM

File: docs/diagrams/ModelClassDiagram.puml [hunk 2]
```diff
@@ -42,6 +43,7 @@ Person *--> Phone
 Person *--> Email
 Person *--> Address
 Person *--> "*" Tag
+Person *--> "*" Group
```
Comment (yimjianbing) [inline docs/diagrams/ModelClassDiagram.puml:46]: Woah, nice use of puml, will have to look into how to use this!

**Review by yimjianbing — APPROVED** [review]
lgtm!!! <3

File: docs/diagrams/FindCommandActivityDiagram.puml
Comment (hyxnnii) [inline docs/diagrams/FindCommandActivityDiagram.puml:1]: Looks like a good skeleton of uml to start with!

**Pull request merged**